apps/aecore/src/aec_peer_connection.erl

```diff
@@ -738,7 +740,7 @@ local_ping_obj(#{ kind := ConnKind, ext_sync_port := Port }) ->
        best_hash    => TopHash,
        difficulty   => Difficulty,
        sync_allowed => ConnKind =/= temporary,
-       share        => 32,  % TODO: make this configurable
+       share        => gossiped_peers(),
```

Comment:
rename to `nr_gossiped_peers`, such that we later know that this actually are not the peers, but a numbeer.

Reply:
what about `gossiped_peers_count`?